frontend/src/components/homepage/pageheader.tsx

```diff
@@ -62,32 +62,32 @@ function Pageheader() {
         </DropdownMenu>
       </div>
 
-      <div className="flex flex-col w-full bg-green-200 fixed bg-opacity-100 z-30 p-4">
+      <div className=" navbar-container flex flex-col w-full fixed bg-opacity-100 z-30 p-4">
```

Comment:
Maybe we could use another brighter color instead as this make our page very dark, we could use a completely different color for the footer background as to make it more pop and attractive like (black for instance)

![Screenshot 2024-05-23 at 7 22 14 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/420c3809-0c17-43b4-a202-689565f68cdc)